website/pages/docs/plugins/sources/hashicorp-vault/_authentication.mdx

```diff
@@ -0,0 +1,7 @@
+The plugin needs to be authenticated in order to sync information from your Vault server.
+
+The plugin requires only _read_ permissions (we will never make any changes to your vault server), so, following the principle of least privilege, it's recommended to grant it read-only permissions.
+
+In Order for CloudQuery to sync resources from your Vault server,
+you will need to obtain a [Vault Token](https://developer.hashicorp.com/vault/docs/concepts/tokens) and configure the `VAULT_TOKEN` environment variable.
```

Comment:
Is the `VAULT_TOKEN` environment variable recognized by Vault tooling in general, or is this special to our plugin? If it's special to plugin, it would be better to create a Spec entry for it

Reply:
`VAULT_TOKEN` is part of the internal vault configuration e.g. see [environment variables](https://developer.hashicorp.com/vault/docs/commands#environment-variables)